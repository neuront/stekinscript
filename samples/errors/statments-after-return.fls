func write(x)
    0

x: 10
if x = 10
    return
    write(x)
write(x = 10)
