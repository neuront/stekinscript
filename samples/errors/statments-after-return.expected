Line: 7
    flow already terminated at Line: 6
